### ISSUE TITLE
cli: timings for `begin transaction` statement are negative

### ISSUE THREAD

**Issue by jordanlewis** [cockroachdb/cockroach#55229]
```
root@:26257/defaultdb> begin transaction;
BEGIN

Time: 0ms total (execution -85636855ms / network 85636855ms)
```

I think this should return something non-negative.

**Comment by arulajmani** [comment]
This should be fixed once we land https://github.com/cockroachdb/cockroach/pull/54991, as the root cause is the same as #54888, #54750.

**Comment by jordanlewis** [comment]
Great thanks! Closing as duplicate of #54888 then.
